js/actions/webviewActions.js

```diff
@@ -81,6 +81,31 @@ const webviewActions = {
         webview.executeJavaScript('document.webkitRequestFullscreen()')
       }
     }
+  },
+
+  findInPage (searchString, caseSensitivity, forward, webview) {
```

Comment:
I actually didn't even know this was valid syntax, but for consistency (and syntax highlighting) I think we should keep the `:`